depends/toolchain.cmake.in

```diff
@@ -153,9 +153,8 @@ else()
 endif()
 
 if("@multiprocess@" STREQUAL "1")
-  set(WITH_MULTIPROCESS ON CACHE BOOL "")
-  set(Libmultiprocess_ROOT "${CMAKE_CURRENT_LIST_DIR}" CACHE PATH "")
-  set(LibmultiprocessNative_ROOT "${CMAKE_CURRENT_LIST_DIR}/native" CACHE PATH "")
+  set(ENABLE_IPC ON CACHE BOOL "")
+  set(Libmultiprocess_EXTERNAL_MPGEN "@depends_prefix@/native/bin/mpgen" CACHE PATH "")
```

Comment:
Please, don't reintroduce more `@depends_prefix@` instances:
```suggestion
  set(Libmultiprocess_EXTERNAL_MPGEN "${CMAKE_CURRENT_LIST_DIR}/native/bin/mpgen" CACHE FILEPATH "")
```

For the context, please see https://github.com/bitcoin/bitcoin/pull/31358.